acceptance/indy_acceptance/indy-tests/test_results/Acceptance_Test_20171117_17-42-58/Test_Scenario_02_Verify_Messages_On_Connection_20171117_17-42-58.log

```diff
@@ -0,0 +1,18 @@
+[95m
```

Comment:
It looks like a log. I think it needs to be removed

Reply:
We removed it.